### PR TITLE
SEO-3239 Windowsforms Page with redirect

### PR DESCRIPTION
**Pull request by Yvone-Atieno**
Hi @Aishwarya-Ganesan ,

Please review the changes for Windowsforms Page with redirect.

Regards,
Yvone.

### CODE REVIEW

File: WindowsForms/layoutmanagers/FlowLayout.md [hunk 1]
```diff
@@ -1241,7 +1241,7 @@ End Sub
...
 
-[Creating a simple Layout](/windowsforms/layoutmanagers/creating-a-simple-layout), [Child Control Settings](/windowsforms/layoutmanagers/layout-manager-settings#child-control-settings),[Centering the Child Controls Horizontally and Vertically](/windowsforms/layoutmanagers/centering-the-child-controls-horizontally-and-vertically), [Rearranging the Controls laid out by GridLayout](/windowsforms/layoutmanagers/gridlayout#rearranging-the-controls-laid-out-by-gridlayout), [Rearranging the Controls laid out by GridBagLayout.](/windowsforms/layoutmanagers/gridbaglayout#rearranging-the-controls-laid-out-by-gridbaglayout)
+[Creating a simple Layout](/windowsforms/layoutmanagers/creating-a-simple-layout), [Child Control Settings](https://help.syncfusion.com/windowsforms/layoutmanagers/layout-manager-settings#child-control-settings),[Centering the Child Controls Horizontally and Vertically](/windowsforms/layoutmanagers/centering-the-child-controls-horizontally-and-vertically), [Rearranging the Controls laid out by GridLayout](/windowsforms/layoutmanagers/gridlayout/overview), [Rearranging the Controls laid out by GridBagLayout.](/windowsforms/layoutmanagers/gridbaglayout/overview)
```
Comment (gowthamanthavasiyappan) [inline WindowsForms/layoutmanagers/FlowLayout.md:1244]: Revert this `https://help.syncfusion.com`

**Review by gowthamanthavasiyappan — CHANGES_REQUESTED** [review]

**Comment by gowthamanthavasiyappan** [comment]
Commented

File: WindowsForms/layoutmanagers/FlowLayout.md (hunk 1 above)
Comment (Yvone-Atieno) [reply WindowsForms/layoutmanagers/FlowLayout.md:1244]: reverted

**Review by gowthamanthavasiyappan — APPROVED** [review]

**Review by Aishwarya-Ganesan — APPROVED** [review]
Changes are fine.

**Pull request merged**